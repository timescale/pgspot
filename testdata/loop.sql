create or replace function l() returns void as $func$
declare
begin
 	loop
 	end loop;
 	loop
 		PERFORM f1('select 1');
    exit when f2();
 	end loop;
end
$func$ language plpgsql;
