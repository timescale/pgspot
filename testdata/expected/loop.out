PS002: Unsafe function creation: l() at line 1
PS005: Function without explicit search_path: l() at line 1
PS016: Unqualified function call: f1 at line 1
PS016: Unqualified function call: f2 at line 1

 Errors: 0 Warnings: 4 Unknown: 0 

